frame/support/src/weights.rs

```diff
@@ -390,7 +395,14 @@ impl sp_runtime::traits::Printable for PostDispatchInfo {
 	fn print(&self) {
 		"actual_weight=".print();
 		match self.actual_weight {
-			Some(weight) => weight.print(),
+			Some(weight) => {
+				"time(".print();
```

Comment:
This type of impl really bad, since each `print` call goes into a separate line. putting a gun at @athei who has implemented `Printable for PostDispatchInfo` and I have been seeing them for a long long time now as multi-line logs when I run a node.

I think both should be fixed, and somehow put into the same line. 

`frame_support::print` might help and it already accepts a formatting.

Reply:
println should be the one that does a new line each. Print should be fine?